semgrep/semgrep/target_manager.py

```diff
@@ -160,17 +161,19 @@ def __str__(self) -> str:
                 f"{len(self.semgrepignored)} files matching .semgrepignore patterns"
             )
         if self.failed_to_analyze:
-            skip_fragments.append(
-                f"{len(self.failed_to_analyze)} files not analyzed due to a parsing or internal Semgrep error"
+            partial_fragments.append(
+                f"{len(self.failed_to_analyze)} files only partially analyzed due to a parsing or internal Semgrep error"
             )
 
-        if not limited_fragments and not skip_fragments:
+        if not limited_fragments and not skip_fragments and not partial_fragments:
             return ""
 
-        message = "Some files were skipped."
+        message = "Some files were skipped or only partially analyzed."
         if limited_fragments:
             for fragment in limited_fragments:
                 message += f"\n  {fragment}"
+        if partial_fragments:
+            message += "\n  Partial Scan: " + ", ".join(partial_fragments)
```

Comment:
```suggestion
            message += "\n  Partially scanned: " + ", ".join(partial_fragments)
```
This makes it sorta-consistent with `Scan skipped` below.